client/landing/stepper/declarative-flow/internals/analytics/record-design.ts

```diff
@@ -92,14 +92,8 @@ export function getDesignEventProps( {
 }
 
 export function getVirtualDesignProps( design: Design ) {
-	/**
-	 * If the design is virtual, and it has a recipe with pattern_ids,
-	 * then we assume that it's a pattern based virtual theme.
-	 */
-	const virtual_theme_pattern = design.is_virtual ? design.recipe?.pattern_ids?.[ 0 ] : null;
-
 	return {
 		is_virtual: design.is_virtual,
-		virtual_theme_pattern,
+		slug: design.is_virtual ? design.recipe?.slug : design.slug,
```

Comment:
Non-blocking: imo, we shouldn't have the slug used for both patterns and themes because it represents two different types of information. It would make it harder to determine which theme is actually used, especially since we'll support multiple themes for virtual themes - we'll change the existing data format. 

I think we should use a new property, like `virtual_theme_pattern`.

Reply:
I wondered about that too, but ultimately decided to stay consistent with how the `slug` property is currently tracked when a style variation or a vertical-generated design is selected.

@ianstewart also suggested if we could have something similar for virtual themes and was open to keep using the same `slug` prop: p1681136519074649/1681135345.566989-slack-C04DZ8M0GHW.

Reply:
Oh, I see. Then if that was the request, then it's fine with me :) I thought that prop should always contain a valid theme and might confuse folks that monitor that event. :)